projects/dotcms-field-elements/src/components/dot-select/dot-select.tsx

```diff
@@ -44,12 +45,28 @@ export class DotSelectComponent {
     _dotTouched = false;
     _dotPristine = true;
 
+    fieldAttr = {
+        type: 'dot-select',
+        name: this.name
+    };
+
     componentWillLoad() {
-        this._options = getDotOptionsFromFieldValue(this.options);
+        this.fieldAttr.name = this.name;
+        this.optionsWatch();
```

Comment:
We need to do the same in all the fields, I think this approach is better because it allows us to scale: https://github.com/dotCMS/core-web/pull/1021/files#diff-efa33eff3d3040ae2246d32cccb56d99R108